BanyanDB: Support custom `TopN pre-aggregation` rules configuration in file `bydb-topn.yml`. (#13319)
apache/skywalking: docs/en/setup/backend/storages/banyandb.md

    shardNum: ${SW_STORAGE_BANYANDB_PROPERTY_SHARD_NUM:1}

```
### TopN Rules Configuration

> The BanyanDB storage supports TopN pre-aggregation in the BanyanDB server side, which trade off more disk_volume as pre-aggregation to save CPU cost, and faster query performance. 

Reviewer: ```suggestion
The BanyanDB storage supports TopN pre-aggregation in the BanyanDB server side, which trades off more disk_volume for pre-aggregation to save CPU cost, and perform faster query in the query stage. 
```
